File: C++/Array plus array.cpp
#include <vector>

int arrayPlusArray(std::vector<int> a, std::vector<int> b)
{
    int sum = 0;
    for (int num : a)
    {
        sum += num;
    }
    for (int num : b)
    {
        sum += num;
    }
    return sum;
}

File: TypeScript/arrayPlusArray.ts
export const arrayPlusArray = (arr1: number[], arr2: number[]): number => {
  let sum1 = arr1.reduce((acc, currVal) => acc + currVal, 0);
  let sum2 = arr2.reduce((acc, currVal) => acc + currVal, 0);
  return sum1 + sum2;
};
